test/shared/data/index.ts

```diff
@@ -0,0 +1,17 @@
+import path from "path"
+import fs from "fs-extra"
+class Data {
+  getPath(name) {
+    return path.join(__dirname, name)
+  }
+
+  getDOMFile(name) {
```

Comment:
Why is `DOM` in the name here? (i.e. why not just `getFile`?)

Reply:
I was trying to distinguish between a [Web Api File](https://developer.mozilla.org/en-US/docs/Web/API/File) class (DOMFile) and a node file, or a brim.file? 

It's not often that we interact with a File instance from the Web APIs so it seemed like calling it out somehow would be useful.

I'm open to a different name though!